focus_areas/code_development.md

```diff
@@ -24,27 +24,65 @@ to time periods.
 
 Goal **Activity**:
 
-* Question **Changes**: How many changes are happening to the code base, during a certain time period? 
+* Question **Changes**: How many changes are happening to the source code, during a certain time period? 
 
-  * Metric **Code_Changes_No**(Period): Number of changes to the code base (see [Code_Changes](../metrics/Code_Changes.md))
-  * Metric **Code_Lines_Changed_No**(Period): Number of lines changed in the source code
-  * Metric **Code_Changes_Lines_No**(Period): Aggregated number of lines touched in all changes.
+  * Metric **Code_Changes**(Period): Number of changes to the source code
+  (see [Code_Changes](../metrics/Code_Changes.md))
+  * Metric **Code_Lines_Changed**(Period): Number of lines changed in the source code.
+  * Metric **Code_Changes_Lines**(Period): Aggregated number of lines touched in all changes.
```

Comment:
I don't think so, since the first is about actual lines changed, while the second is about total number of lines in the patches... But this is getting confusing (I got some more comments about this). I'm removing the one about lines changed, which is more difficult to measure, too, and stay only with the total number of lines, aggregating those in patches. We can later recover the other one, if we feel it is important.

Reply:
:+1: